src/v2/Utils/scrollHelpers.ts

```diff
@@ -24,3 +26,19 @@ export const scrollIntoView = (props: ScrollIntoViewProps) => {
     })
   }
 }
+
+export const useScrollIntoView = (isRouteActive: boolean) => {
+  const ref = React.useRef() as any
+  React.useEffect(() => {
+    if (isRouteActive && ref?.current) {
```

Comment:
For something generic like `scroll thing into view` whats the purpose of `isRouteActive`?